### PR TITLE
Bmi Dot indicator component

### PR DESCRIPTION
**Pull request by chiragVDario**

### CODE REVIEW

File: dariobmibar/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -10,4 +10,11 @@
     <color name="bmi_section_1">#e1efe6</color>
     <color name="bmi_section_2">#fff2e3</color>
     <color name="bmi_section_3">#fcdada</color>
+
+    <color name="bmi_dot_indicator_1">#F16658</color>
+    <color name="bmi_dot_indicator_2">#07CB79</color>
+    <color name="bmi_dot_indicator_3">#FFCC4D</color>
+    <color name="bmi_dot_indicator_4">#F16658</color>
+    <color name="black">#000000</color>
+    <color name="grey">#6f676f</color>
```
Comment (raphael-dario) [inline dariobmibar/src/main/res/values/colors.xml]: wrong color for the text. please look in figma for the right color code.

File: dariobmibar/src/main/res/layout/dario_bmi_dot_indicator.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            app:layout_constraintTop_toTopOf="@id/bar1"
+            app:layout_constraintBottom_toBottomOf="@id/bar1"
+            android:layout_marginBottom="34dp" />
+
+        <TextView
+            android:id="@+id/bmiText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            tools:text="Healthy"
+            android:textSize="14sp"
```
Comment (raphael-dario) [inline dariobmibar/src/main/res/layout/dario_bmi_dot_indicator.xml]: wrong size for the text, please look at the figma design.

File: dariobmibar/src/main/java/com/labstyle/dariobmibar/DarioBMIDotIndicator.kt [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                R.id.bmiText,
+                ConstraintSet.END,
+                R.id.selectionLineIndicator,
+                ConstraintSet.END
+            )
+        }
+
+        constraintSet.applyTo(parent)
+
+        val circleBg =
```
Comment (raphael-dario) [inline dariobmibar/src/main/java/com/labstyle/dariobmibar/DarioBMIDotIndicator.kt:93]: it's the same drawable for all cases here.

**Review by raphael-dario — CHANGES_REQUESTED** [review]
it doesn't seem to work. if I set a bmi of 18.0, the dot should be at the end of the 1st section.

File: dariobmibar/src/main/java/com/labstyle/dariobmibar/DarioBMIDotIndicator.kt (hunk 3 above)
Comment (chiragVDario) [reply dariobmibar/src/main/java/com/labstyle/dariobmibar/DarioBMIDotIndicator.kt:93]: yes

File: dariobmibar/src/main/res/values/colors.xml (hunk 1 above)
Comment (chiragVDario) [reply dariobmibar/src/main/res/values/colors.xml]: changes done

File: dariobmibar/src/main/res/layout/dario_bmi_dot_indicator.xml (hunk 2 above)
Comment (chiragVDario) [reply dariobmibar/src/main/res/layout/dario_bmi_dot_indicator.xml]: changes done

**Review by raphael-dario — APPROVED** [review]

**Pull request merged**